plugins/kubernetes/src/components/DeploymentTables/DeploymentTables.tsx

```diff
@@ -0,0 +1,221 @@
+      const containerStatuses = pod.status?.containerStatuses ?? [];
+      const containersReady = containerStatuses.filter(cs => cs.ready).length;
+
+      return `${containersReady}/${containerStatuses.length}`;
+    },
+  },
+  {
+    title: 'total restarts',
+    render: (pod: V1Pod) => {
+      const containerStatuses = pod.status?.containerStatuses ?? [];
+      return containerStatuses?.reduce((a, b) => a + b.restartCount, 0);
+    },
+    type: 'numeric',
+  },
+  {
+    title: 'status',
+    width: '20%',
+    render: (pod: V1Pod) => {
+      const containerStatuses = pod.status?.containerStatuses ?? [];
+      const errors = containerStatuses.reduce((accum, next) => {
+        if (next.state === undefined) {
+          return accum;
+        }
+
+        const waiting = next.state.waiting;
+        const terminated = next.state.terminated;
+
+        const renderCell = (reason: string | undefined) => (
+          <Fragment key={`${pod.metadata?.name}-${next.name}`}>
+            <SubvalueCell
+              value={<StatusError>Container: {next.name}</StatusError>}
+              subvalue={reason}
+            />
+            <br />
+          </Fragment>
+        );
+
+        if (waiting) {
+          accum.push(renderCell(waiting.reason));
+        }
+
+        if (terminated) {
+          accum.push(renderCell(terminated.reason));
+        }
+
+        return accum;
+      }, [] as React.ReactNode[]);
+
+      if (errors.length === 0) {
+        return <StatusOK>OK</StatusOK>;
+      }
+
+      return errors;
+    },
+  },
+  {
+    title: 'Pod Initialized',
+    align: 'center',
+    render: (pod: V1Pod) => {
+      const conditions = pod.status?.conditions ?? [];
+      return renderCondition(conditions.find(c => c.type === 'Initialized'));
+    },
+  },
+  {
+    title: 'Pod Ready',
+    align: 'center',
+    render: (pod: V1Pod) => {
+      const conditions = pod.status?.conditions ?? [];
+      return renderCondition(conditions.find(c => c.type === 'Ready'));
+    },
+  },
+  {
+    title: 'Containers Ready',
+    align: 'center',
+    render: (pod: V1Pod) => {
+      const conditions = pod.status?.conditions ?? [];
+      return renderCondition(
+        conditions.find(c => c.type === 'ContainersReady'),
+      );
+    },
+  },
+  {
+    title: 'Pod Scheduled',
+    align: 'center',
+    render: (pod: V1Pod) => {
+      const conditions = pod.status?.conditions ?? [];
+      return renderCondition(conditions.find(c => c.type === 'PodScheduled'));
+    },
+  },
+];
+
+type DeploymentTablesProps = {
+  deploymentTriple: DeploymentTriple;
+  children?: React.ReactNode;
+};
+
+export const DeploymentTables = ({
+  deploymentTriple,
+}: DeploymentTablesProps) => {
+  const isOwnedBy = (
+    ownerReferences: V1OwnerReference[],
+    obj: V1Pod | V1ReplicaSet | V1Deployment,
+  ): boolean => {
+    return ownerReferences?.some(or => or.name === obj.metadata?.name);
+  };
+
+  return (
+    <Grid
+      style={{ width: '100%' }}
```

Comment:
Shouldn't be needed should it? ^_>

We don't want this to be needed at least :grin:

Reply:
Must have forgot to remove 😉